src/PrestaShopBundle/Controller/Admin/Improve/Design/CmsPageController.php

```diff
@@ -225,6 +225,7 @@ public function createAction(Request $request)
         return $this->render(
             '@PrestaShop/Admin/Improve/Design/Cms/add.html.twig',
             [
+                'cmsPageId' => 0,
```

Comment:
Just a suggestion but you could get rid of this in the controller, and in the twig template use `'data-id': cmsPageId|default(0)`

Just to keep the controller cleaner and to make sure the form rendering is resillient

Reply:
Thanks @jolelievre, it's done. I've no time to lean twig for now but I'll do it.